test/test-regexes.js

```diff
@@ -0,0 +1,54 @@
+const assert = require('assert');
+
+const bcd = require('..');
+
+function lookup(dottedFeature) {
+  const x = dottedFeature.split('.');
+  const feature = x.reduce((prev, current) => prev[current], bcd);
+  return feature;
+}
+
+function testToken(feature, matches, misses) {
+  const str = feature.__compat.matches.regex_token || feature.__compat.matches.regex_value;
+  const regexp = new RegExp(str);
+
+  matches.forEach(match => assert.ok(regexp.test(match), `${regexp} did not match ${match}`));
+  misses.forEach(miss => assert.ok(!regexp.test(miss), `${regexp} erroneously matched ${miss}`));
+}
+
+const tests = [
```

Comment:
These will some day be in separate files, I hope.

Reply:
Yeah, I thought about splitting them from the outset, but I figured it'd be more meaningful to split them up once we have more than two examples to work from.

(For similar reasons, I didn't use one of the JS testing frameworks like Jest—I figured it wouldn't be worth adding a dependency until we can show a need for it.)